scalatest/src/main/scala/org/scalatest/tools/Runner.scala

```diff
@@ -1580,7 +1581,7 @@ object Runner {
         }
       }
 
-      new URLClassLoader(urlsList.toArray, classOf[Suite].getClassLoader)
+      new URLClassLoader(urlsList.toArray.asInstanceOf[Array[URL | Null]], classOf[Suite].getClassLoader)
```

Comment:
**Pass array into java method**
Pass `Array[T]` as argument to java method that requires an array. Java arrays have type `Array[T | Null]`.